analysis_driver/dataset.py

```diff
@@ -468,22 +466,21 @@ def non_useable_run_elements(self):
 
     @property
     def project_id(self):
-        return self.run_elements[0]['project_id']
+        return self.sample['project_id']
 
     def _amount_data(self):
-        return sum(
-            [
-                int(r.get(ELEMENT_NB_Q30_R1_CLEANED, 0)) + int(r.get(ELEMENT_NB_Q30_R2_CLEANED, 0))
-                for r in self.run_elements
-            ]
-        )
+        return int(self.sample.get('aggregated').get('clean_yield_in_gb')*1000000000)
```

Comment:
The casting to `int` here is pretty useless. Check if `self.sample.get('aggregated').get('clean_yield_in_gb')` returns an `int` and if it does not then cast it

Reply:
When I checked this it returned a float, which is why it is cast to an int here

Reply:
That makes sense